src/replicate/getDocs.js

```diff
@@ -53,7 +54,11 @@ function getDocs(src, diffs, state) {
         bulkGetInfo.docs.forEach(function (doc) {
           if (doc.ok) {
             resultDocs.push(doc.ok);
+          } else if(typeof doc.error !== 'undefined') {
```

Comment:
`doc.error !== undefined` seems to be necessary.  I don't recall why right now, but can investigate if you're interested.